server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/LeaguePointsCalculationService.java

```diff
@@ -0,0 +1,89 @@
+package de.tum.in.www1.hephaestus.leaderboard;
+
+import de.tum.in.www1.hephaestus.gitprovider.pullrequest.PullRequest;
+import de.tum.in.www1.hephaestus.gitprovider.user.User;
+import java.time.OffsetDateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LeaguePointsCalculationService {
+    private final Logger logger = LoggerFactory.getLogger(LeaguePointsCalculationService.class);
+
+    public int calculateNewPoints(User user, LeaderboardEntryDTO entry) {
+        // Initialize new players with 1000 points
```

Comment:
Wouldn't it generally make more sense to have all the numbers as "magic numbers" in some dictionary or just on top of the code, so it is easier to change those if required.

Reply:
Very fair point! I extracted all constants into static variables now, which should be easier to change on demand. See 1219d97